analyzers/its/expected/ManuallyAddedNoncompliantIssues.CS/S4487-NetFramework48.json

```diff
@@ -1,11 +1,5 @@
 {
   "Issues": [
-    {
-      "Id": "S4487",
-      "Message": "Remove this unread private field \u0027name\u0027 or refactor the code to use its value.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/NetFramework48/InsecureDeserialization.cs#L50",
```

Comment:
Wasn't this a valid issue? I don't understand why it dissapeared.

Reply:
That looks correct to me. S4487 and S1144 share the same analyzer class.
[The example](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/NetFramework48/InsecureDeserialization.cs#L50) shows a class that has a private member that can be initialized through the constructor. Then the class can be serialized, so the private member is actually used/read. So I consider this a former FP.